.github/workflows/ci_upgrade.yml

```diff
@@ -0,0 +1,162 @@
+name: JDBC Upgrade Tests
```

Comment:
Can we rename this test specify that will test minor version upgrade? i.e. minor-version-upgrade.yml
So that later we can add another test as major-version-upgrade.yml to test MVU.

Reply:
Done. Renamed it to Minor Version Upgrade tests.